contants.ts

```diff
@@ -0,0 +1,12 @@
+export const VIEW_WIDGET = "widget";
```

Comment:
@zhik did you mean to call this file `constants.ts` rather than "contants?"

Reply:
yea :S